pkg/virt-controller/watch/export/export.go

```diff
@@ -454,21 +457,21 @@ func (ctrl *VMExportController) handlePodSucceededOrFailed(vmExport *exportv1.Vi
 func (ctrl *VMExportController) handlePVCsForVirtualMachineSnapshot(vmExport *exportv1.VirtualMachineExport, vmSnapshot *snapshotv1.VirtualMachineSnapshot) ([]*corev1.PersistentVolumeClaim, int, error) {
 	var content *snapshotv1.VirtualMachineSnapshotContent
 	var err error
-	pvcs := make([]*corev1.PersistentVolumeClaim, 0)
+	pvcs := []*corev1.PersistentVolumeClaim{}
```

Comment:
This declaration confuses me.  It is basically equivalent to what you are replacing?  Why is 0 length slice more desirable than nil slice?

I hate to keep talking about this, but I think zero length slices should be avoided and equivalent declarations should not be mixed.  Basically `var s []sliceType` all the things unless there is a good reason to do otherwise

Reply:
So my java roots are showing, I didn't realize you can append a nil slice and it just works, I was expecting an npe and was defending against it. I looked it up and you can append nil slice no problem.